Jenkinsfile

```diff
@@ -74,26 +74,26 @@ try {
           mkdir clang-debug-unity-odr && cd clang-debug-unity-odr &&                                                   ${cmake}           ${debug}          ${clang} ${unity} -DCMAKE_UNITY_BUILD_BATCH_SIZE=0 .. &\
-          mkdir clang-debug-addr-ub-sanitizers && cd clang-debug-addr-ub-sanitizers &&                                 ${cmake} ${ccache} ${debug}          ${clang}          -DENABLE_ADDR_UB_SANITIZATION=ON .. &\
-          mkdir clang-release-addr-ub-sanitizers && cd clang-release-addr-ub-sanitizers &&                             ${cmake} ${ccache} ${release}        ${clang}          -DENABLE_ADDR_UB_SANITIZATION=ON .. &\
-          mkdir clang-release && cd clang-release &&                                                                   ${cmake} ${ccache} ${release}        ${clang}          .. &\
-          mkdir clang-release-addr-ub-sanitizers-no-numa && cd clang-release-addr-ub-sanitizers-no-numa &&             ${cmake} ${ccache} ${release}        ${clang}          -DENABLE_ADDR_UB_SANITIZATION=ON -DENABLE_NUMA_SUPPORT=OFF .. &\
-          mkdir clang-relwithdebinfo-thread-sanitizer && cd clang-relwithdebinfo-thread-sanitizer &&                   ${cmake} ${ccache} ${relwithdebinfo} ${clang}          -DENABLE_THREAD_SANITIZATION=ON .. &\
-          mkdir clang-relwithdebinfo-thread-sanitizer-no-numa && cd clang-relwithdebinfo-thread-sanitizer-no-numa &&   ${cmake} ${ccache} ${relwithdebinfo} ${clang}          -DENABLE_THREAD_SANITIZATION=ON  -DENABLE_NUMA_SUPPORT=OFF .. &\
+          mkdir clang-debug-addr-ub-sanitizers && cd clang-debug-addr-ub-sanitizers &&                                 ${cmake}           ${debug}          ${clang}          -DENABLE_ADDR_UB_SANITIZATION=ON .. &\
+          mkdir clang-release-addr-ub-sanitizers && cd clang-release-addr-ub-sanitizers &&                             ${cmake}           ${release}        ${clang}          -DENABLE_ADDR_UB_SANITIZATION=ON .. &\
+          mkdir clang-release && cd clang-release &&                                                                   ${cmake}           ${release}        ${clang}          .. &\
+          mkdir clang-relwithdebinfo-thread-sanitizer && cd clang-relwithdebinfo-thread-sanitizer &&                   ${cmake}           ${relwithdebinfo} ${clang}          -DENABLE_THREAD_SANITIZATION=ON .. &\
           mkdir gcc-debug && cd gcc-debug &&                                                                           ${cmake} ${ccache} ${debug}          ${gcc}            .. &\
           mkdir gcc-release && cd gcc-release &&                                                                       ${cmake} ${ccache} ${release}        ${gcc}            .. &\
           wait"
         }
 
         parallel clangDebug: {
           stage("clang-debug") {
-            sh "export CCACHE_BASEDIR=`pwd`; cd clang-debug && make all -j \$(( \$(cat /proc/cpuinfo | grep processor | wc -l) / 4)) && ../scripts/analyze_ccache_usage.py"
```

Comment:
Aren’t you afraid we won’t put the analayze script back in once the cmake bug is fixed?

Reply:
Doesn't really matter. That was just for me to check things in the output of Jenkins. If we put ccache back in and forget it, the worst thing that could happen is someone missing and adding it.